### ISSUE TITLE
Strange behavior with rana theme

### ISSUE THREAD

**Issue by igordeo-costa** [ohmybash/oh-my-bash#248]
I'm facing some trouble with the Rana theme that I'm not having with other themes I tested (purity, agnoster, powerline, rjorgenson, roderik) and I don't know if it is to be expected: when I press arrow up to navigate through the history, at some moment the command line stays frozen or it shows half of the previous line and I can't access the command I want.

For instance: suppose I have typed the following commands, in that exact order:

```
nano ~/.bashrc
latexmk -help
cd /home/dados/Acadêmicos
``` 
If I try to go back (arrow up) to open the .bashrc file, this is what I find:

One arrow up (OK):
```
cd /home/dados/Acadêmicos
``` 
Two arrows up (but I can't run the latexmk command):

```
cd /home/dados/Acadêmicoslatexmk -help
```
Three arrows up:
```
cd /home/dados/Acadêmicosnano ~/.bashrc
```
In other words, the command line stays paralyzed in the first command and I can't run the others.

This behavior does not appear with the other themes I tested.

I'm running bash in Ubuntu 20.04.

I'm sorry if it is a silly question, but I don't know how to solve this problem.

**Comment by einwickler** [comment]
I am also running bash in Ubuntu 20.04 (using Gnome Terminal) and just reprodued this issue in the vanilla rana theme.

I just had the exact same issue after making some modifications to the agnoster theme (that's why I stumbled upon this issue in the first place). I figured out it was caused by some not properly formatted ANSI Escape color codes I inserted. So maybe it's worth a try to search for places where color codes are inserted.
